arc/src/arc_aio.h

```diff
@@ -3,6 +3,9 @@
 #ifndef __arc_aio_h__
 #define __arc_aio_h__
 
+// ZJS includes
+#include "zjs_ipm.h"
+
```

Comment:
Really arc_aio.h should include that then... that's what my `checkheaders` script tries to check for but I must not run it on arc headers.